components/SearchBar.tsx

```diff
@@ -22,7 +23,21 @@ export default function SearchBar({
         placeholder="Search for your favorite records"
         className="grow pl-10 py-2 rounded-xl text-black"
         onChange={(e) => setUserInput(e.target.value)}
+        value={userInput}
       />
+      <button
+        onClick={() => {
```

Comment:
if you want to focus the input after it's a bit more tricky because you need to use the DOM API's. [Here's a page on the react docs about using useRef to manipulate DOM elements.](https://react.dev/learn/manipulating-the-dom-with-refs)

when you click this button you want to:
1. clear the userInput state (you're already doing this)
2. focus the input element (the page I linked above has an example of this)